config/crd/bases/databases.digitalocean.com_databaseclusters.yaml

```diff
@@ -15,7 +15,17 @@ spec:
     singular: databasecluster
   scope: Namespaced
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
+    - jsonPath: .spec.engine
+      name: Engine
+      type: string
+    - jsonPath: .spec.name
+      name: DO name
```

Comment:
Need a `make manifests` to update this, I believe :-)

Reply:
Sorry, I forgot this. Updated now.